cunumeric/config.py

```diff
@@ -433,6 +434,61 @@ class CuNumericTunable(IntEnum):
     HAS_NUMAMEM = _cunumeric.CUNUMERIC_TUNABLE_HAS_NUMAMEM
 
 
+# Match these to BitGeneratorOperation in cunumeric_c.h
+@unique
+class BitGeneratorOperation(IntEnum):
+    CREATE = _cunumeric.CUNUMERIC_BITGENOP_CREATE
+    DESTROY = _cunumeric.CUNUMERIC_BITGENOP_DESTROY
+    RAND_RAW = _cunumeric.CUNUMERIC_BITGENOP_RAND_RAW
+    DISTRIBUTION = _cunumeric.CUNUMERIC_BITGENOP_DISTRIBUTION
+
+
+# Match these to BitGeneratorType in cunumeric_c.h
+@unique
+class BitGeneratorType(IntEnum):
+    DEFAULT = _cunumeric.CUNUMERIC_BITGENTYPE_DEFAULT
+    XORWOW = _cunumeric.CUNUMERIC_BITGENTYPE_XORWOW
+    MRG32K3A = _cunumeric.CUNUMERIC_BITGENTYPE_MRG32K3A
+    MTGP32 = _cunumeric.CUNUMERIC_BITGENTYPE_MTGP32
+    MT19937 = _cunumeric.CUNUMERIC_BITGENTYPE_MT19937
+    PHILOX4_32_10 = _cunumeric.CUNUMERIC_BITGENTYPE_PHILOX4_32_10
+
+
+# Match these to BitGeneratorDistribution in cunumeric_c.h
+@unique
+class BitGeneratorDistribution(IntEnum):
+    INTEGERS_32 = _cunumeric.CUNUMERIC_BITGENDIST_INTEGERS_32
```

Comment:
a nitpick: maybe we should sort distributions alphabetically so that we can later spot them quickly?

Reply:
can we push that until we have all of them ? merge would be easier that way.